packages/genesys-spark-components/src/components/stable/gux-tooltip/gux-tooltip.tsx

```diff
@@ -219,7 +219,9 @@ export class GuxTooltip {
           }}
           data-placement={this.placement}
         >
-          <slot />
+          <slot name="content">
```

Comment:
Would changing the slot to have a name be considered a breaking change? will it break apps that currently use it with no slot name?

Reply:
Yes but by falling back to a default slot in the new named slot I think it makes it a non breaking change.